.eslintrc.json

```diff
@@ -1,6 +1,9 @@
 {
   "extends": "airbnb",
   "root": true,
+  "env": {
+    "browser": true
```

Comment:
An alternative to this is to explicitly name the browser globals when you use them, at the top of the file:

/* global window */

I tend to prefer this approach so that nothing slips past, but in a sample I could go either way.